File: Source/TurnTracker.Client/src/app/auth/models/Session.ts
import { DateTime } from 'luxon';

export class Session {
  public name: string;
  public created: string;
  public updated: string;
  public id: number;
  public current: boolean;

  // Properties populated by the client and not the server
  public createdDate: DateTime;
  public updatedDate: DateTime;
}

File: Source/TurnTracker.Domain/Models/Session.cs
using System;

namespace TurnTracker.Domain.Models
{
    public class Session
    {
        public string Name { get; set; }
        public DateTimeOffset Created { get; set; }
        public DateTimeOffset Updated { get; set; }
        public long Id { get; set; }
        public bool Current { get; set; }
    }
}